crates/astria-sequencer/src/app/action_handler.rs

```diff
@@ -1,14 +1,9 @@
 use cnidarium::StateWrite;
 
-/// This trait is a verbatim copy of `cnidarium_component::ActionHandler`.
```

Comment:
Deleted because this didn't seem like a super accurate representation of `ActionHandler` anymore, but can revert if others disagree.